validitymap.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package amppackager
+
+import (
+	"net/http"
+)
+
+type ValidityMap struct {
+	validityMap []byte
+}
+
+func NewValidityMap() (*ValidityMap, error) {
+	this := new(ValidityMap)
+	this.validityMap = []byte("\xA0")
```

Comment:
Leave a comment that this is the empty map, and link to https://tools.ietf.org/html/draft-yasskin-httpbis-origin-signed-exchanges-impl-00#section-3.6.

Reply:
Done